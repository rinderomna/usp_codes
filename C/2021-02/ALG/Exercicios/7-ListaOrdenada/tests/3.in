INSERE 1 Luis Carlos
INSERE 2 Fernanda Montenegro
INSERE 4 Einstein
INSERE 3 Mickey
REMOVE 5
INSERE 11 Joao Cesar
INSERE 7 Sett
IMPRIMIR

