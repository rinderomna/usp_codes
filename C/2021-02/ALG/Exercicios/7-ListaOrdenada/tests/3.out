INVALIDO
1, Luis Carlos; 2, Fernanda Montenegro; 3, Mickey; 4, Einstein; 7, Sett; 11, Joao Cesar;
